src/main/java/appeng/parts/automation/PartAnnihilationPlane.java

```diff
@@ -359,7 +359,36 @@ private boolean handleOverflow( EntityItem entityItem, IAEItemStack overflow )
 		entityItem.getEntityItem().stackSize = newStackSize;
 
 		return changed;
+	}
+
+	/**
+	 * Spawns an overflow item as new {@link EntityItem} into the {@link World}
+	 *
+	 * @param w the world to spawn it
+	 * @param x x coordinate
+	 * @param y y coordinate
+	 * @param z coordinate
+	 * @param overflow the item to spawn
+	 */
+	private void spawnOverflow( IAEItemStack overflow )
+	{
+		if( overflow == null )
+		{
+			return;
+		}
```

Comment:
Probably fine for this case, I usually just prefer handling this stuff at least also on the lowest level and not have to wrap every call of `PartAnnihilationPlane.spawnOverflow` with a null check. Having one single null check everything needs to go through simply is a bit less code.

I could change it to a `PreCondition` or `assert`. But that then throw an exception under some rare conditions. Also I cannot use assertions in all cases. Like enabled assertions will cause Extra Utilities to not start indev, which I needed to test it this.